projects/go-lib/src/lib/components/go-datepicker/year-view/go-calendar-year-view.component.ts

```diff
@@ -60,22 +60,18 @@ export class GoCalendarYearViewComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.setUpYears(this.year.value - 15);
+    this.setUpYears(this.year.value - 6);
     this.setFocusedYear(this.year.value);
   }
 
   public nextYearGroup(): void {
     this.setUpYears(this.lastYear.value + 1);
-    this.setFocusedYear(this.focusedYear.value + 24);
+    this.setFocusedYear(this.focusedYear.value + (3 * 4));
```

Comment:
Is there a reason to do this `(3 * 4)` calculation and not just put `12`?

Reply:
nope